docs/root/configuration/http_conn_man/headers.rst

```diff
@@ -438,6 +438,11 @@ route, virtual host, and/or global route configuration level. See the relevant :
 <config_http_conn_man_route_table>` and :ref:`v2 <envoy_api_msg_RouteConfiguration>` API
 documentation.
 
+The *:path* and *:authority* headers may not be modified via this mechanism,
+instead mechanisms such as :ref:`prefix_rewrite
+<envoy_api_field_route.RouteAction.prefix_rewrite>` and :ref:`host_rewrite
+<envoy_api_field_route.RouteAction.host_rewrite>` should be used.
```

Comment:
Do we want to generally extend this to all the colon-prefixed pseudo-headers? I imagine changing some of the others is dangerous as well. Otherwise, lgtm.

Reply:
That's a good question. Are there legitimate case to want to rewrite `:method` or `:scheme`? I think `:scheme` would be rewritten anyway depending on upstream, so probably not, not sure about `:method` CC @alyssawilk

Reply:
There's also `:no-chunks` and `:status`.